### ISSUE TITLE
Retain flags ?

### ISSUE THREAD

**Issue by HeikoMueller** [soft2tec/universal_mqtt_client#21]
Hi, 
nice library that works smoothly with Flutter! Just a question for clarification: Is it possible to set a "retain: true" flag with this library?  
Thank you!

**Comment by lucacasonato** [comment]
No, that is currently not possible. It should be a simple addition to make though.
